### DISCUSSION TITLE
Export_solver

### DISCUSSION THREAD

**Discussion by Randolph03** [phoebe-project/phoebe2#918] [Questions]
Hello, I would like to ask, I run phoebe's estimator on my pc, the continued the optimizer (nm) by using export solver (from input data, upto the estimator n optimizer, all are in one same file in proper ordee). I didn't specifically set the initial of nm, but does it automatically take result from lcgeom estimator and use it as initial for nm, especially when using export_solver? Thank you very much

**Comment by kecnry** [comment]
If `initial_values` is not provided, then the optimizer starts at the face values of the parameters.  Solvers propose changes to parameter values, but do not change the face-values of parameters until they are adopted.  So if you called `b.adopt_solution(...)` after the lcgeom estimator, then it will take those values as a starting place, yes.  But if you only called `b.run_solver(...)` for lc geometry but never adopted changes, then it will not.

For more information see:
* [General solver tutorial](http://phoebe-project.org/docs/latest/tutorials/solver)
* [nelder_mead API docs](http://phoebe-project.org/docs/latest/api/phoebe.parameters.solver.optimizer.nelder_mead)
* [adopt_solution API docs](http://phoebe-project.org/docs/latest/api/phoebe.frontend.bundle.Bundle.adopt_solution)
